instructions/conventional-commit.prompt.md

```diff
@@ -0,0 +1,80 @@
+---
+Description: "Framework for generating conventional commit messages using a structured XML format. This prompt guides users through the process of creating standardized, descriptive commit messages in accordance with the Conventional Commits specification."
+Version: "1.0.0"
+Created: "2025-07-25"
+---
```

Comment:
It might be worth adding a `tools` section to specify that it requires terminal access